src/Razor/src/Microsoft.CodeAnalysis.Remote.Razor/NoOpTelemetryReporter.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Immutable;
+using Microsoft.AspNetCore.Razor.Telemetry;
+
+namespace Microsoft.CodeAnalysis.Remote.Razor;
+
+internal class NoOpTelemetryReporter : ITelemetryReporter
```

Comment:
nit: I'm not sure why we need two of these?

Reply:
Oops, you're right. I think this can just go in Razor.Workspaces and get shared between here and rzls